extensions/EditComments/template/en/default/hook/global/header-start.html.tmpl

```diff
@@ -16,10 +16,13 @@
   js_BUGZILLA.string.InlineCommentEditor = {
     cancel => 'Cancel',
     cancel_tooltip => 'Discard the changes',
+    edit => 'Edit',
     edited => 'Edited',
     fetch_error => 'Raw comment could not be loaded. Please try again later.',
     hide_revision => 'Hide This Revision',
     loading => 'Loading…',
+    preview => 'Preview',
+    preview_error = 'Preview could not be loaded. Please try again later.',
```

Comment:
Note to myself: next time I edit this file, replace this `=` with `=>`. Perl is... 😕

Reply:
Not perl, template toolkit trying to be helpful